### DISCUSSION TITLE
Aurora Lighting AU-A1ZBDSS

### DISCUSSION THREAD

**Discussion by oneofthemany** [Koenkk/zigbee2mqtt#17264] [Devices]
@Koenkk 

the wiki says that the device does not support read power draw when it does:

![image](https://user-images.githubusercontent.com/44924639/230413831-1194c0a4-2910-4624-ba6e-d218dae4469b.png)

potentially update the wiki to enable end users to be aware of feature support
![image](https://user-images.githubusercontent.com/44924639/230414760-445ac2e9-84a8-4624-b2e8-42f4b4a0d7af.png)

![image](https://user-images.githubusercontent.com/44924639/230415376-3e7ab1ad-0757-49a4-9732-88e7beb8ecd9.png)

:)

**Comment by Koenkk** [comment]
Did you try the `/get` command? (can you provide the debug log when doing this?)

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable debug logging.

**Reply by oneofthemany** [reply]
Totally forgot about this, sorry I'm mistaken with my comment
